### ISSUE TITLE
Is there a way to use inverse with Websockets?

### ISSUE THREAD

**Issue by krisiye** [igniterealtime/openfire-inverse-plugin#55]
Was wondering if there was an option for configuring the defaultEndpoint for the plugin based on an env var or similar? Reviewing this https://github.com/igniterealtime/openfire-inverse-plugin/blob/ec6ea101dc842225552cd63ab9fb2cc51f2fb549/src/java/org/igniterealtime/openfire/plugin/inverse/ConfigServlet.java#LL50C22-L50C37 it looks like we only support http-bind (per the browser request scheme/server/port).  I was trying to get this to use a secured websocket URI.  Any suggestions?

**Comment by guusdk** [comment]
I think you can override that default using the `inverse.config.bosh_service_url` property, if I look at this bit of the code: https://github.com/igniterealtime/openfire-inverse-plugin/blob/ec6ea101dc842225552cd63ab9fb2cc51f2fb549/src/java/org/igniterealtime/openfire/plugin/inverse/ConfigServlet.java#L221-L223

**Comment by krisiye** [comment]
@guusdk  a follow up question to plugin configuration. I verified the above works when set from the admin console and system properties (which persists the settings to ofproperty). Is there a way to do this without being added manually? I tried the following within openfire.xml but does not seem to be working:

```xml
<jive>
  <prop>
    <inverse>
      <config>
        <bosh_service_url>wss://localhost/ws</bosh_service_url>
      </config>
    </inverse>
  </prop>
</jive>
```

Any suggestions on how i could have openfire migrate the property to the db on bootstrap?
